Jenkinsfile

```
@@ -52,8 +52,10 @@ node("docker") {
         if ("$EXECUTION_MODE".toString().equals("Publish packages")) {
             buildRpmAndDeb(version, architectures)
 
+            // Download cert files, to be used for signing the Windows executable, packaged by Chocolatey.
+            downloadToolsCert()
             stage('Build and Publish Chocolatey') {
-                publishChocoPackage(version,jfrogCliRepoDir)
+                publishChocoPackage(version, jfrogCliRepoDir, architectures)
```

I'm not sure it is necessary
```suggestion
                publishChocoPackage(version, jfrogCliRepoDir)
```

---

The ```publishChocoPackage``` function does not have access to the "global" ```architectures``` variable.
I also thought it does initially, but it doesn't.